test/integration/etl_test.py

```diff
@@ -68,6 +68,17 @@ def test_ferc1_xbrl2sqlite(ferc1_engine_xbrl, ferc1_xbrl_taxonomy_metadata):
     )
 
 
+class TestCsvExtractor:
+    """Verify that we can lead CSV files as provided via the datastore."""
+
+    def test_extract_eia176(self, pudl_datastore_fixture):
```

Comment:
Followed the preexisting pattern for this (e.g., per `test_extract_eia860`). I was able to quickly and successfully run this spot check in isolation with
```
 pytest test/integration/etl_test.py -k test_extract_eia176
```